examples/example2/style.css

```diff
@@ -0,0 +1,128 @@
+.wrapper {
+  padding: 10px;
+}
+
+.form {
+  max-width: 500px;
+  margin: 0 auto;
+  padding: 20px;
+  border-radius: 10px;
+  border: 1px solid rgba(0, 0, 0, 0.1);
+  background-color: #fff;
+}
+
+.form-fields {
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-gap: 20px;
+}
+
+.form-group {
+  position: relative;
+}
+
+/* Position the label over the input field (as if its a placeholder) */
+.label {
```

Comment:
Why not position it `top: 0; left: 0;` and set the height to 100%? No need for magic numbers 🙂

Reply:
Hm that is easier said than done unfortunately. Setting it to 100% would use the height of its relative parent, which also contains the error message and some other margins. So that is not exactly the height of the mollie-component.

But I agree that getting rid of the magic numbers would be nice.. will think about a better solution